src/app/components/challenge/challenge.component.html

```diff
@@ -1,42 +1,105 @@
-      <div class="challenge-img-parent">
-        <div class="challenge-img-wrap">
-          <img class="challenge-img" [src]="challenge['image']" (error)="challenge['image']='assets/images/404.png';" />
+<div class="mid-container">
+    <div class="challenge-container">
+        <div class="challenge-card">
+            <div class="top-card-container ev-card-panel">
+                <div class="row card-img-row">
+                    <div class="col-lg-2 col-md-3 col-sm-12">
+                        <div class="image-container">
+                            <img [src]="challenge['image']" width="100%">
+                        </div>
+                    </div>
+                    <div class="col-lg-7 col-md-6">
+                        <div class="card-content">
+                            <p>
+                                <span style="line-height: 1.5" class="fs-18">
+                                    <strong style="color: rgba(0, 0, 0, 0.87);" class="fw-semibold">
+                                        {{challenge['title']}}
+                                    </strong>
+                                </span>
+                                &nbsp;
+                                <span *ngIf="isChallengeHost">
+                                    <a style="padding-right: 4px;" class="pointer" (click)="editChallengeTitle()">
+                                        <i class="fa fa-pencil" aria-hidden="true"></i>
+                                    </a>
+                                    <a class="pointer" (click)="deleteChallenge()">
+                                        <i class="fa fa-trash red-text" aria-hidden="true"></i>
+                                    </a>
+                                </span>
+                                <br>
+                                <span style="line-height: 1.5" *ngIf="challenge.creator.team_url != ''">
+                                    <strong class="text-light-black fw-semibold fs-15">Organized by:</strong>
+                                    <a class="orange-text" href="{{challenge['creator']['team_url']}}" target="_blank"> 
+                                        {{challenge['creator']['team_name']}} 
+                                    </a>
+                                </span>
+                                <span style="line-height: 1.5" *ngIf="challenge.creator.team_url == ''">
+                                    <strong class="text-light-black fw-semibold fs-15">Organized by:</strong>
+                                     {{challenge['creator']['team_name']}} 
+                                </span>
+                                <br>
+                                <span style="line-height: 1.5" *ngIf="isChallengeHost">
+                                    <span><a class="pointer" (click)="togglePublishChallengeState()">
+                                        <strong class="text-light-black fw-semibold fs-15">{{publishChallenge.state}} </strong>
```

Comment:
Are you not using any HTML prettify plugin in text editor? Plugins can do all these formattings automatically for you. Also please maintain line length < 80 (plugin is there for this as well).  HTML tags can be used line by line to make it readable.

Reply:
@Shekharrajak how can I make HTML tags line by line for line length < 80 as it contains so many attributes inside it, instead I can wrap to be like attributes by attributes